src/olympia/reviewers/tests/test_commands.py

```diff
@@ -129,6 +129,7 @@ def test_fetch_candidates(self):
         DiscoveryItem.objects.create(
             recommendable=True, addon=recommended_addon)
 
+        # ---------------------------------------------------------------------
```

Comment:
🤔

Reply:
Yeah... This test method is getting huge and I needed a way to separate the different setup parts... They could be extracted to separate functions, but those would only be useful for this particular test so I dunno...

Reply:
I wasn't sure if it was something you added during debugging because the line locations seemed a bit arbitrary at first glance.  130 lines of setup is a bit much... but I guess extracting some to separate functions wouldn't really make it more understandable either (perhaps less so).  🤷‍♂